sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedRegionInfo.java

```diff
@@ -40,7 +40,7 @@ final class EncryptedRegionInfo implements JsonSerializable<EncryptedRegionInfo>
      *
      * @return The ciphertextLength property.
      */
-    public int getDataLength() {
+    public long getDataLength() {
```

Comment:
This would be a breaking change. Is this necessary? I note in many places it is cast back down to `int`.

Reply:
This class is internal only, so it wouldn't be affecting customers. Also in the long run, we plan to use **long** instead of **int** everywhere to allow customers to have the ability to use larger ranges (greater than 2GB, which is the max supported by ints).

Reply:
nit: this class in package-private. So, we should consider moving this to `implementation` package so that it's easy to review. It's easier to notice a class in implementation package than to know whether it's a public or package-private class in such git diffs.

Reply:
We will move all internal classes to the implementation package in a follow-up PR to keep functional changes separate from this PR.